src/Screens/pomodoro-app-screen.js

```diff
@@ -9,17 +9,20 @@ import { IoCheckmarkDoneCircleSharp } from "react-icons/io5";
 import { useNavigate } from "react-router-dom";
 import { Button } from "../components";
 import { useData } from "../helper";
+import { EmptyPage } from "./empty-page";
 import "./screen.css";
 const PomoDoroApp = ({ setTask, setIsModalOpen }) => {
   const { todoState, dispatchToDo } = useData();
-  const { todo } = todoState;
+  const { todo, searchTerm } = todoState;
   const [selectedTag, setSelectedTag] = useState("All");
+  console.log(searchTerm);
```

Comment:
cleanup `console.log()`

Reply:
Thanks for pointing that out. Cleaning it